README.md

```diff
@@ -13,19 +13,19 @@ dependencies: ["https://github.com/DataDog/serverless-plugin-datadog/blob/master
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue)](https://github.com/DataDog/serverless-plugin-datadog/blob/master/LICENSE)
 
 Datadog recommends the Serverless Framework Plugin for developers using the Serverless Framework to deploy their serverless applications.
-The plugin automatically enables instrumentation for your Python and Node.js applications to collect metrics, traces, and logs by:
+The plugin automatically enables instrumentation for your Python, Node.js, Ruby, Java, Go, and .NET applications to collect metrics, traces, and logs by:
 
 - Installing the Datadog Lambda library to your Lambda functions as a Lambda layer.
 - Installing the Datadog Lambda Extension to your Lambda functions as a Lambda layer (`addExtension`) or subscribing the Datadog Forwarder to your Lambda functions' log groups (`forwarderArn`).
-- Making the required configuration changes, such as adding environment variables, to your Lambda functions.
+- Making the required configuration changes, such as adding environment variables or additional tracing layers (.NET), to your Lambda functions.
```

Comment:
```suggestion
- Making the required configuration changes, such as adding environment variables or additional tracing layers (such as .NET), to your Lambda functions.
```

Reply:
I would actually suggest dropping the parenthetical completely, I don't think we need to specify .NET here.